src/main/java/org/minions/devfund/marcos/uml/MapSite.java

```diff
@@ -0,0 +1,22 @@
+package org.minions.devfund.marcos.uml;
+
+/**
+ * It is in charged to represent a Map Site object.
+ */
+public abstract class MapSite {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Convert the abstract class "MapSite" into an interface. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1610)